speech/cloud-client/transcribe_dlp.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Google Cloud Speech API sample that demonstrates using DLP
+to automatically discover and redact sensitve data.
+
+Example usage:
+    python transcribe_dlp.py deidentify -p <project_id>
+    python transcribe_dlp.py deidentify
+                            -f './resources/sallybrown.flac' -p <project_id>
+"""
+
+import argparse
+import io
+import re
+
+# Imports the Google Cloud client library
+from google.cloud import speech
+from google.cloud import dlp
+from google.cloud.speech import enums
+from google.cloud.speech import types
+
+
+def deidentify(file_name, project_id):
+    """ Convert speech to text and then de-identify entities using DLP. """
+    # Instantiates a client
+    speech_client = speech.SpeechClient()
+
+    dlp_client = dlp.DlpServiceClient()
+
+    parent = dlp_client.project_path(project_id)
+
+    # Prepare info_types by converting the list of strings into a list of
+    info_types = ['ALL_BASIC']
+    # dictionaries (protos are also accepted).
+    inspect_config = {
+        'info_types': [{'name': info_type} for info_type in info_types]
+    }
+    # Construct deidentify configuration dictionary
```

Comment:
This comment is also unhelpful - it is a literal redefinition of the following code. Can we update to explain why or what the user is doing exactly?